inspirehep/dojson/hep/fields/bd5xx.py

```diff
@@ -85,32 +85,37 @@ def hidden_note2marc(self, key, value):
 
 
 @hep.over('thesis', '^502..')
-@utils.for_each_value
```

Comment:
Luckily there are no records on legacy with more than on 502 field, because otherwise you would have had a `list` of `dict`s rather than a `dict` in `value`, by removing this decorator.